SCons/Tool/install.py

```diff
@@ -48,64 +46,112 @@
 class CopytreeError(EnvironmentError):
     pass
 
-# This is a patched version of shutil.copytree from python 2.5.  It
-# doesn't fail if the dir exists, which regular copytree does
-# (annoyingly).  Note the XXX comment in the docstring.
-def scons_copytree(src, dst, symlinks=False):
-    """Recursively copy a directory tree using copy2().
 
-    The destination directory must not already exist.
-    If exception(s) occur, an CopytreeError is raised with a list of reasons.
+def scons_copytree(src, dst, symlinks=False, ignore=None, copy_function=copy2,
+                   ignore_dangling_symlinks=False, dirs_exist_ok=False):
+    """Recursively copy a directory tree, SCons version.
+
+    This is a modified copy of the Python 3.7 shutil.copytree function.
+    SCons update: dirs_exist_ok dictates whether to raise an exception in
+    case dst or any missing parent directory already exists. Implementation
+    depends on os.makedirs having a similar flag, which it has since
+    Python 3.2.  This version uses a change from Python 3.8, so at some
```

Comment:
Can you add a 
`#TODO Remove when minimum python is 3.8 or higher`
Above this function?